areas/models.py

```diff
@@ -36,13 +36,11 @@ class ContractZone(models.Model):
     contractor = models.CharField(
         verbose_name=_("contractor"), max_length=255, blank=True
     )
-    contractor_user = models.ForeignKey(
+    contractor_users = models.ManyToManyField(
```

Comment:
you do not need to migrate any data, right?

Reply:
Yup no need, there are no actual users yet.